forge/test/mlir/test_ops_tf.py

```diff
@@ -66,14 +71,14 @@ def call(self, x):
             return self.conv2d(x)
 
     inputs = [tf.random.uniform((batch_size, input_height, input_width, input_channels), dtype=activations_dtype)]
-    
+
     framework_model = Conv2d()    
     fw_out = to_pt_tensors(framework_model(*inputs))
 
     compiled_model = forge.compile(framework_model, sample_inputs=inputs)
     co_out = compiled_model(*inputs)
+
     co_out = [co.to("cpu").to(fw_out[0].dtype) for co in co_out]
-      
     assert compare_tensor_to_golden("conv2d", fw_out[0], co_out[0].reshape(fw_out[0].shape))
 
 def test_dual_conv2d():
```

Comment:
@LPanosTT comment when you take a look around this issue:
- https://github.com/tenstorrent/tt-forge-fe/issues/281

Let's set `dtype` as pytest input params, just to be able to test different DF for dual convs as well